debian/control

```diff
@@ -30,8 +30,7 @@ Depends:
     python3 (>= 3.5),
     python3-dbus,
     python3-gi,
-    python3-taglib,
-    xdg-utils
+    python3-taglib
```

Comment:
Was this intentional in this PR? It doesn't really seem to have anything to do with the rest of your commits?

Reply:
It's intentional. The only tool we used in the xdg-utils package is xdg-open, which is now replaced with GIO's launch_default_for_uri method.